src/app/components/mnoe-users-local-list/mnoe-users-local-list.coffee

```diff
@@ -92,21 +93,21 @@
           MnoErrorsHandler.processServerError(error)
       ).finally(-> user.isUpdatingEmail = false)
 
-    scope.keyFromRole = (role) ->
-      _.find(USER_ROLES, 'value', role).label
-
-    scope.updateUserRole = (user, oldRole) ->
+    scope.updateUserRole = (organization, user, oldRole) ->
```

Comment:
Why pass `organization` and not keep `scope.organization`?
The editbox is only visible when `scope.organizationContext` is true

Reply:
This was so that this `#scope.updateUserRole` took the same arguments as the `#scope.updateUserRole` in `mnoe-organizations-local-list.coffee`. The latter needs an organization as an argument, because there are multiple possible organizations.I can change back to previous implementation if wanted.

Reply:
Yes please.
No need to keep the method signature as these are two different components.
`MnoeUsers.updateUserRole` has the same signature for both cases so all good